conf/taiga/conf.json.j2

```diff
@@ -1,7 +1,7 @@
 {
     "api": "{{ TAIGA_SCHEME }}://{{ TAIGA_HOSTNAME }}/api/v1/",
     "eventsUrl": null,
-    "debug": true,
+    "debug": {% if DEBUG.lower() == 'true' %}true{% else %}false{% endif %},
```

Comment:
I think it could be just `{{ DEBUG|lower }}` or `{{ DEBUG.lower() }}` - you can check which one works :wink:

Reply:
It's my suggestion, I will not reject it because of this. But it would be nice... :smile:

Reply:
I'll try it out.

Reply:
Thanks :blush: